testers/testers/markus_test_specs.py

```diff
@@ -10,10 +10,24 @@ def from_json(cls, json_str):
         return cls(json.loads(json_str))
 
     def __getitem__(self, key):
-        return self._specs[key]
+        try:
+            return self._specs[key]
+        except KeyError:
+            if isinstance(key, tuple):
+                d = self
+                for k in key:
+                    d = d[k]
+                return d
```

Comment:
What is this tuple access necessary for?

Reply:
I'll add some documentation to explain